frontend/src/containers/ChatToolbar/index.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Routine } from 'redux-saga-routines';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  IconDefinition,
+  faLock,
+  faHashtag,
+  faPodcast,
+  faAt,
+  faCopy,
+  faSearch,
+  faIdCard,
+  faTh,
+  faDatabase,
+  faReply,
+  faPlus,
+  faPlay
+} from '@fortawesome/free-solid-svg-icons';
+import { IAppState } from '../../common/models/store';
+import { IWorkspaceState } from '../../scenes/Workspace/reducers/reducer';
+import { IChat } from '../../common/models/workstate/chat';
+import styles from './styles.module.sass';
+import { userChannelRoutine } from '../../scenes/Workspace/routines/routines';
+
+let linkPanel = false;
+let directPanel = false;
+
+interface IProps {
+  workstate: IWorkspaceState;
+  userChannelRoutine: Routine;
```

Comment:
see how it's implemented in other places

Reply:
Done